plugins/exec.lua

```diff
@@ -11,6 +12,30 @@ local function exec(cmd, keep_newline)
 end
 
 
+local function get_active_view()
+  if getmetatable(core.active_view) == DocView then
+    return core.active_view
+  end
+  return nil
+end
```

Comment:
No need for this function as you get the active `DocView` from the command predicate, so you can pass it directly to the other functions.